src/interfaces.ts

```diff
@@ -13,6 +13,7 @@ interface CalendarEvent {
   busy?: boolean;
   guests?: string[];
   url?: string;
+  status?: 'CONFIRMED' | 'TENTATIVE' | 'CANCELLED';
```

Comment:
I prefer [string literal unions over enums](https://www.typescriptcourse.com/string-literal-unions-over-enums), but happy to switch this to an enum if that's what is preferred for this codebase